src/app/CASESessionManager.cpp

```diff
@@ -77,12 +77,12 @@ void CASESessionManager::ReleaseSession(PeerId peerId)
     ReleaseSession(FindExistingSession(peerId));
 }
 
-void CASESessionManager::ReleaseSessionsForFabric(CompressedFabricId compressedFabricId)
+void CASESessionManager::ReleaseSessionsForFabric(CompressedFabricId compressedFabricId) const
```

Comment:
This is not conceptually a const method, right?  Its current implementation happens to be const because the device pool is pointed to, not an actual member, but that's an implementation detail.